application/classes/Ushahidi/Repository/Tos.php

```diff
@@ -0,0 +1,76 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi Tos Repository
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2017 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Entity\Tos;
+use Ushahidi\Core\Entity\TosRepository;
+use Ushahidi\Core\SearchData;
+use Ushahidi\Core\Traits\UserContext;
+
+
+
+class Ushahidi_Repository_Tos extends Ushahidi_Repository implements
+    TosRepository
+{
+    use UserContext;
+
+
+    // Ushahidi_Repository
+    protected function getTable()
+    {
+        return 'tos';
+    }
+
+
+    // CreateRepository
+    public function create(Entity $entity)
+    {
+        //get the user ID
```

Comment:
For some reason the linter was getting mad with spaces (only sometimes) after the // in comments. Will try to fix and see if it works this time.